services/web/server/src/simcore_service_webserver/director_v2_core.py

```diff
@@ -211,6 +211,21 @@ async def create_or_update_pipeline(
         log.error("could not create pipeline from project %s: %s", project_id, exc)
 
 
+@log_decorator(logger=log)
+async def is_pipeline_running(
+    app: web.Application, user_id: PositiveInt, project_id: UUID
+) -> bool:
+
+    # TODO: make it cheaper by /computations/{project_id}/state
+    pipeline = await get_computation_task(app, user_id, project_id)
+    if pipeline is None:
+        # the pipeline might not exist and that is ok
+        # FIXME:  or some error happened so ... we assume is not running??
```

Comment:
Maybe you do not care about errors in this case? You just correctly reply it is not running. I'd remove the FIXME.